includes/class-sensei-settings.php

```diff
@@ -840,6 +849,48 @@ private function get_changed_setting_values( $field, $new_value, $old_value ) {
 
 		return array_filter( array_merge( $added, $removed ) );
 	}
+
+	/**
+	 * Learning mode setting.
+	 *
+	 * @param array $args The field arguments.
+	 */
+	public function render_learning_mode_setting( $args ) {
```

Comment:
Small thing, I would move this function to a separate, with HTML it feels a bit out of place here

Reply:
I agree, looks a little odd on it's own, because it's the only html in this file. But this class (`Sensei_Settings`) extends the `Sensei_Settings_API`, which is all about rendering the plugin settings markup. This method simply adds an edge case that is not handled by the `Sensei_Settings_API` class.